enderio/src/datagen/java/com/enderio/enderio/datagen/common/loot/EIOBlockLootProvider.java

```diff
@@ -24,15 +29,20 @@
 
 public class EIOBlockLootProvider extends BlockLootSubProvider {
     public EIOBlockLootProvider(HolderLookup.Provider registries) {
-        super(Set.of(), FeatureFlags.DEFAULT_FLAGS, registries);
+        super(Set.of(), FeatureFlagSet.of(FeatureFlags.VANILLA, EIOFeatureFlags.ENDERFACE, EIOFeatureFlags.NIARD, EIOFeatureFlags.FARMING_STATION), registries);
     }
 
     @Override
     protected Iterable<Block> getKnownBlocks() {
-        return EIOBlocks.BLOCKS.getEntries()
+        var eioBlocks = EIOBlocks.BLOCKS.getEntries()
             .stream()
             .map(e -> (Block) e.value())
             .toList();
+        var machineBlocks = EIOBlocks.BLOCKS.getEntries()
```

Comment:
This is a hangover from when EIOBlocks and MachineBlocks existed separately - remove this change.